### PR TITLE
Log out button

### PR DESCRIPTION
**Pull request by MarianaCarlota**
I putted it in the edit page. In the dashboard it was not looking good. I also added a button to go back. it didn't had one.

image

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -64,6 +68,10 @@
       <% end %>
     </div>
 
+    <div class="logout">
+      <i class="fas fa-door-open"></i><%= link_to " Log out", destroy_user_session_path, data: { "turbo-method": :delete } %>
+    </div>
```
Comment (wJoenn) [inline app/views/devise/registrations/edit.html.erb:73]: Remove the div and include the icon inside the link please

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <div class="container">
   <div class="devise">
+    <%= link_to dashboard_path, class: "btn-home" do %>
+      <i class="fa-solid fa-arrow-left"></i>
+    <% end %>
```
Comment (wJoenn) [inline app/views/devise/registrations/edit.html.erb:5]: You could be using the same back button as for the other forms for consistency

**Review by wJoenn — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (MarianaCarlota) [reply app/views/devise/registrations/edit.html.erb:5]: but I am. Isn't it this one?

Comment (MarianaCarlota) [reply app/views/devise/registrations/edit.html.erb:5]: image

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (MarianaCarlota) [reply app/views/devise/registrations/edit.html.erb:73]: I'm getting this error. What am I donig wrong?

image

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (MarianaCarlota) [reply app/views/devise/registrations/edit.html.erb:5]: oh, the other forms, you mean. Sure we can keep that one

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (wJoenn) [reply app/views/devise/registrations/edit.html.erb:69]: You need to remove the "Log Out" and move it inside the do end

**Review by wJoenn — APPROVED** [review]

**Pull request merged**